django_spanner/client.py

```diff
@@ -9,5 +9,16 @@
 
 
 class DatabaseClient(BaseDatabaseClient):
+    """Wrap the Django base class."""
+
     def runshell(self, parameters):
+        """Override the base class method.
+
+        TODO: Consider actual implementation of this method.
```

Comment:
It's not clear that we could implement this since cloud spanner doesn't have a "shell" executable. We might want to consider https://github.com/cloudspannerecosystem/spanner-cli or a similar project for this in the future.

Reply:
I removed the docstring here, just raising `NotSupportedError` is fine since spanner doesn't really give us a way to support this.